week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,9 +10,17 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
-
+  constructor() {
+    this.cart = [];
+  }
+  addItem(item) {
+    this.cart.push(item);
+  }
+  listItems() {
+    this.cart.forEach((item) => console.log(item));
+  }
   cartContains() {
+    console.log(this.cart);
```

Comment:
You are very close with this one but you are missing some things from your console log, have another look at what the question says the output should be.

Great job!